### DISCUSSION TITLE
E_MISSING_ENV_VALUE: Missing environment variable "HOST" after build

### DISCUSSION THREAD

**Discussion by wemersonrv** [adonisjs/core#3513] [Help] [answered]
Hello guys, i'm having an issue when running the app builded version. It gives the error:

>  E_MISSING_ENV_VALUE: Missing environment variable "HOST"

Steps executed:
```bash
% npm run build
% cd build
% npm ci --production 
% node server.js
```
![image](https://user-images.githubusercontent.com/2028673/151080123-222180ff-72ca-4608-8094-1575b5d342c8.png)

Well... I'm pretty sure the .env and the Env.ts files had it:
![image](https://user-images.githubusercontent.com/2028673/151080411-1fab669b-6683-46d3-b962-259fa0880981.png)

What am I doing wrong?

**Comment by Julien-R44** [comment, marked as answer]
You have to copy the .env file in the "build" folder. Or load your environment variables at OS level

**Reply by wemersonrv** [reply]
> You have to copy the .env file in the "build" folder. Or load your environment variables at OS level

Ah. Understand. I thought that when make the build it reads and convert .env it into code.

OK thanks. I copied and it worked!
